src/ifcopenshell-python/ifcopenshell/api/alignment/__get_mapped_segments.py

```diff
@@ -0,0 +1,60 @@
+# IfcOpenShell - IFC toolkit and geometry engine
+# Copyright (C) 2025 Thomas Krijnen <thomas@aecgeeks.com>
+#
+# This file is part of IfcOpenShell.
+#
+# IfcOpenShell is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# IfcOpenShell is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with IfcOpenShell.  If not, see <http://www.gnu.org/licenses/>.
+
+import ifcopenshell
+import ifcopenshell.api.alignment
+from ifcopenshell import entity_instance
+from typing import Sequence
+
+
+def __get_curve_segment_count(segment: entity_instance) -> int:
+    """
+    returns the number of IfcCurveSegment that an IfcAlignmentSegment maps to.
+    generally this is a 1 to 1 mapping, with helmert curve being the exception
+    """
+    if segment.DesignParameters.is_a("IfcAlignmentHorizontalSegment"):
+        return 2 if segment.DesignParameters.PredefinedType == "HELMERTCURVE" else 1
+    elif segment.DesignParameters.is_a("IfcAlignmentVerticalSegment"):
+        return 1
+    elif segment.DesignParameters.is_a("IfcAlignmentCantSegment"):
+        return 2 if segment.DesignParameters.PredefinedType == "HELMERTCURVE" else 1
+
+
+def __get_mapped_segments(file: ifcopenshell.file, segment: entity_instance) -> Sequence[entity_instance]:
```

Comment:
```suggestion
def __get_mapped_segments(file: ifcopenshell.file, layout_segment: entity_instance) -> Sequence[entity_instance]:
```

Unfortunately we don't have any sort of entity typing on the python side so it is helpful to be precise with variable names.

Reply:
good call - revised as noted